### PR TITLE
Moved getch to scripts

### PR DESCRIPTION
**Pull request by deanwetherby**

### CODE REVIEW

File: machine_common_sense/__init__.py [hunk 1]
```diff
@@ -19,7 +19,6 @@
 from .history_writer import HistoryWriter
 from .step_metadata import StepMetadata
 from .util import Util
-from .getchHelper import getch
```
Comment (deanwetherby) [inline machine_common_sense/__init__.py:22]: There's no need for getch in the MCS package as it is only used for the human input script.

File: scripts/run_human_input.py [hunk 2]
```diff
@@ -3,6 +3,8 @@
 
 import machine_common_sense as mcs
 
+from getchHelper import getch
```
Comment (deanwetherby) [inline scripts/run_human_input.py:6]: importing getch directly from the module rather than mcs namespaced.

Comment (ThomasSchellenbergNextCentury) [inline scripts/run_human_input.py:6]: While we're at it, should we rename this file to `getch_helper.py` to follow python conventions?

Comment (deanwetherby) [reply scripts/run_human_input.py:6]: Bam! Absolutely

Comment (rartiss55) [reply scripts/run_human_input.py:6]: I think getchHelper needs to be updated to getch_helper since it got renamed?

**Comment by brianpippin** [comment]
Do we want to update the getch_helper file to use our python naming conventions as well or is that file just something we are using?

**Comment by deanwetherby** [comment]
> Do we want to update the getch_helper file to use our python naming conventions as well or is that file just something we are using?

In what way?

**Comment by brianpippin** [comment]
> > Do we want to update the getch_helper file to use our python naming conventions as well or is that file just something we are using?
> 
> In what way?

I think I was looking at class names, and forgot those can be camelcase, so feel free to just ignore my comment.

**Review by kdrumm — APPROVED** [review]

**Pull request merged**